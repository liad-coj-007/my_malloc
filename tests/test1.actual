[10, 20, 30, 40, 50]
Total free bytes: 20
Total used bytes: 20
Number of free blocks: 1
Number of allocated blocks: 1
