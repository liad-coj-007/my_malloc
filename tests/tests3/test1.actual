[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
                               Memory Statistics
                               =================
Free blocks: 32
Free bytes: 4127744
Allocated blocks: 33
Allocated bytes: 4193248
Meta data bytes: 1056
                               Memory Statistics
                               =================
Free blocks: 32
Free bytes: 4193280
Allocated blocks: 32
Allocated bytes: 4193280
Meta data bytes: 1024
