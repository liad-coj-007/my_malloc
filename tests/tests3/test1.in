add 1 16376
put 1 0 1
put 1 1 2
put 1 2 3
put 1 3 4
put 1 4 5
put 1 5 6
put 1 6 7
put 1 7 8
put 1 8 9
put 1 9 10
print 1 0 9
print_stats
free 1
print_stats