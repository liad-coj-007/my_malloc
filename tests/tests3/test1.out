[1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
                               Memory Statistics
                               =================
Free blocks: ME 
Free bytes: MH - ME - 65504
Allocated blocks: ME + 1
Allocated bytes: MH - ME 
Meta data bytes: ME*ME+ME
                               Memory Statistics
                               =================
Free blocks: ME
Free bytes: MH
Allocated blocks: ME
Allocated bytes: MH 
Meta data bytes: ME*ME